gulpfile.js

```diff
@@ -249,7 +252,14 @@ gulp.task("regen", ["jekyll"], function () {
 });
 
 gulp.task("fetch", function (done) {
-    exec("node", [bin("fetch_docs.js"), FETCH_CONFIG, FETCH_DIR], done);
+    if (!argv.nofetch) {
```

Comment:
This could also just check if the fetch directory exists. If you want to re-fetch, just run `gulp clean`.